plugins/transformations/sequence2array/srcCxx/Sequence2ArrayTransformation.cxx

```diff
@@ -194,7 +172,7 @@ bool Seq2ArrayTransformation::are_type_compatible(
         }
 
         if (input_array_type.total_element_count() != output_array_type.total_element_count()) {
```

Comment:
I think the check should be for `input_array_type.total_element_count() > output_array_type.total_element_count()`